src/wp-admin/js/widgets/media-widgets.js

```diff
@@ -632,7 +632,7 @@ wp.mediaWidgets = ( function( $ ) {
 		 */
 		syncModelToInputs: function syncModelToInputs() {
 			var control = this;
-			document.querySelectorAll( '.media-widget-instance-property' ).forEach( function( input ) {
+			control.syncContainer.querySelectorAll( '.media-widget-instance-property' ).forEach( function( input ) {
```

Comment:
To fix the broken QUnit tests I think this line may need to be:
`control.syncContainer[0]. etc`

Reply:
No, `control.syncContainer[0]` is not defined. `control.syncContainer` is already the JS object.

The typical reason for adding `[0]` in these files is to get the JS object from a jQuery object, but there is no jQuery object here. Might the reason for the broken tests be precisely that it's now dealing with a JS object instead of a jQuery object?

Reply:
Is it to do with the way the testing is set up here then:
`tests/qunit/wp-admin/js/widgets/test-media-video-widget.js`?

Reply:
Could be. It looks like these lines might be the issue (and they're replicated in the tests for the other media files too):
```
videoWidgetControlInstance = new wp.mediaWidgets.controlConstructors.media_video({
	el: jQuery( '<div></div>' ),
	syncContainer: jQuery( '<div></div>' ),
	model: videoWidgetModelInstance
});
```
I'm not sure how to change `jQuery( '<div></div>' )` to regular JS, though.